src/Mailgun/Api/Pagination.php

```diff
@@ -22,12 +22,52 @@
     abstract protected function safeDeserialize(ResponseInterface $response, $className);
 
     /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationNext($response)
+    {
+        return $this->getPaginationUrl($response->getNextUrl(), get_class($response));
+    }
+
+    /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationPrevious($response)
```

Comment:
Well, that is why I suggested `previousPage()` for example, but in the other hand, we are working with a Pagination object so make sense to use a method name like `previous()`. Feel free to choose, as far it works as expected.